src/main/resources/modules/total_joint_replacement/functional_status_assessments.json

```diff
@@ -0,0 +1,1141 @@
+        "http://www.koos.nu",
+        ""
+      ]
+    },
+    "PROMIS10_Total_Assessment": {
+      "type": "MultiObservation",
+      "category": "survey",
+      "number_of_observations": 2,
+      "codes": [
+        {
+          "system": "LOINC",
+          "code": "61576-5",
+          "display": "PROMIS short form - global - version 1.1"
+        }
+      ],
+      "direct_transition": "PROMIS10_Attribute_Set",
+      "remarks": [
+        "Two different subscales are scored separately: Global Mental Health and Global Physical Health. Each have a score between 4 - 20."
+      ]
+    },
+    "VR12_Total_Assessment_2": {
+      "type": "MultiObservation",
+      "category": "survey",
+      "number_of_observations": 8,
+      "codes": [
+        {
+          "system": "LOINC",
+          "code": "71934-4",
+          "display": "Veterans Rand health survey - 12 item (VR-12)"
+        }
+      ],
+      "direct_transition": "VR12_Attribute_Set_2",
+      "remarks": [
+        "Eight different subscales are scored separately: Mental Health, Role Emotion, Social Functioning, Vitality, General Health, Bodily Pain, Role Physical, and Physical Functioning. Each have a score between 4 - 20."
+      ]
+    },
+    "PROMIS10_Total_Assessment_2": {
+      "type": "MultiObservation",
+      "category": "survey",
+      "number_of_observations": 2,
+      "codes": [
+        {
+          "system": "LOINC",
+          "code": "61576-5",
+          "display": "PROMIS short form - global - version 1.1"
+        }
+      ],
+      "direct_transition": "PROMIS10_Attribute_Set_2",
+      "remarks": [
+        "Two different subscales are scored separately: Global Mental Health and Global Physical Health. Each have a score between 4 - 20."
+      ]
+    },
+    "HOOS_Total_Assessment": {
+      "type": "MultiObservation",
+      "category": "survey",
+      "number_of_observations": 5,
+      "codes": [
+        {
+          "system": "LOINC",
+          "code": "72092-0",
+          "display": "Hip Dysfunction and Osteoarthritis Outcome Score [HOOS]"
+        }
+      ],
+      "direct_transition": "HOOS_Attribute_Set",
+      "remarks": [
+        "Five different subscales are scored separately: HOOS Pain (9 items); HOOS Symptoms (7 items); HOOS ADL (17 items); HOOS Sport/Rec (5 items); HOOS QOL (4 items). The scores are transformed to a 0–100 scale, with zero representing extreme knee problems and 100 representing no knee problems.",
+        "http://www.koos.nu"
+      ]
+    },
+    "Joint_Replacement_Assessment_Terminal": {
+      "type": "Terminal",
+      "conditional_transition": [
```

Comment:
Please remove the transitions from the Terminal state -- terminals can't have transitions. (I'm actually surprised the UI allowed it, and it doesn't show on the graphical view so it's easy to miss)

Reply:
Didn't even realize it was there - got it.